src/hooks/use-site-details.tsx

```diff
@@ -22,6 +22,7 @@ interface SiteDetailsContext {
 		siteName?: string,
 		wpVersion?: string,
 		customDomain?: string,
+		enableSSL?: boolean,
 		callback?: ( site: SiteDetails | void ) => Promise< void >
```

Comment:
```suggestion
		callback?: ( site: SiteDetails ) => Promise< void >
```
Per my other suggestion of updating the callback signature.